### PR TITLE
Only give the name 'VPC' to an instance if the account supports EC2 Classic

### PR DESCRIPTION
**Pull request by emmasax1**
Description and Impact
----------------------
With newer AWS accounts, the accounts do not support EC2 classic anymore, just VPC. So, if the account doesn't support EC2 classic, then we have no need for including the word 'VPC' in the title of the instance. So, add in a check for that, so that newer accounts' instance names will match up correctly.

Deploy Plan
-----------
> Does Platform Operations need to know anything special about this deploy? Are migrations present?

Rollback Plan
-------------
* To roll back this change, revert the merge with: `git revert -m 1 MERGE_SHA` and perform another deploy.

URLs
----
http://docs.aws.amazon.com/sdkforruby/api/Aws/EC2/Client.html#describe_account_attributes-instance_method

QA Plan
-------
- [x] Test out in either Slack or terminal and make sure all of the numbers are now adding up correctly
- [x] Verify that names of instances are now matching up as they should
- [x] Verify the word 'VPC' doesn't appear where it shouldn't
- [x] Run an account that does support EC2 classic and verify those numbers are staying the same

### CODE REVIEW

File: lib/sport_ngin_aws_auditor/ec2_instance.rb [hunk 1]
```diff
@@ -52,14 +62,17 @@ def get_more_info(instances, client)
       private :get_more_info
     end
 
-    attr_accessor :id, :name, :platform, :availability_zone, :scope, :instance_type, :count, :stack_name, :tag_value, :tag_reason, :expiration_date, :count_remaining
-    def initialize(ec2_instance, tag_name, count=1)
+    attr_accessor :id, :name, :platform, :availability_zone, :scope, :instance_type, :count, :stack_name, :tag_value, :tag_reason, :expiration_date, :count_remaining, :ec2_classic_support
```
Comment (bbergstrom) [inline lib/sport_ngin_aws_auditor/ec2_instance.rb]: http://sportngin.github.io/styleguide/ruby.html#source-code-layout_L17

You should change this super long single line into a more readable multi-line.

File: lib/sport_ngin_aws_auditor/ec2_instance.rb [hunk 2]
```diff
@@ -52,14 +62,17 @@ def get_more_info(instances, client)
...
+    def initialize(ec2_instance, tag_name, ec2_classic_support, count=1)
```
Comment (bbergstrom) [inline lib/sport_ngin_aws_auditor/ec2_instance.rb]: Is creating a new argument here necessary?  It looks like your initialize method could call the new ec2_classic_support method directly to get the value.

File: lib/sport_ngin_aws_auditor/ec2_instance.rb [hunk 3]
```diff
@@ -16,17 +16,27 @@ def get_instances(client=AWS.ec2, tag_name=nil)
...
       def get_retired_reserved_instances(client)
         client.describe_reserved_instances.reserved_instances.map do |instance|
           next unless instance.state == 'retired'
-          new(instance, nil, instance.instance_count)
+          new(instance, nil, ec2_classic_support(client), instance.instance_count)
         end.compact
       end
 
+      def ec2_classic_support(client=AWS.ec2)
+        attr_vals = client.describe_account_attributes.account_attributes.first.attribute_values
```
Comment (bbergstrom) [inline lib/sport_ngin_aws_auditor/ec2_instance.rb]: Could we be more specific here and inspect the values for `supported-platforms` attribute?

**Comment by emmasax1** [comment]
### QA: Everything matches up as it should:
screen shot 2017-03-31 at 10 27 49 am

**Pull request merged**